src/components/RichTextEditor/Plugins/TextStylePlugin/TextStyleDropdown/index.tsx

```diff
@@ -11,8 +11,9 @@ import { TextStyleDropdownProps } from './types';
 
 export const TextStyleDropdown = ({ editorId, textStyles = defaultTextStyles }: TextStyleDropdownProps) => {
     const [open, setOpen] = useState(false);
-    const editor = usePlateEditorState(editorId)!;
-    const { designTokens } = useRichTextEditorContext();
+    const editor = usePlateEditorState(editorId);
+    const context = useRichTextEditorContext();
+    const designTokens = context.designTokens as Partial<Record<TextStyles, CSSProperties>>;
```

Comment:
```suggestion
    const { designTokens } = useRichTextEditorContext();;
```
The type should already be `DesignTokens`

Reply:
fixed